docs/public/GettingHelp.md

```diff
@@ -0,0 +1,15 @@
+# Getting Help
+
+- Refer to [opentelemetry.io](https://opentelemetry.io/) for general
+  information about OpenTelemetry.
+- Refer to the
+  [OpenTelemetry Dotnet GitHub repository](https://github.com/open-telemetry/opentelemetry-dotnet)
+  for further information and resources related to OpenTelemetry Dotnet.
+- Feel free to join the
+  [CNCF OpenTelemetry Dotnet Slack channel](https://cloud-native.slack.com/archives/C01N3BC2W7Q).
+  If you are new, you can create a CNCF Slack account
+  [here](http://slack.cncf.io/). Ask any questions related to OpenTelemetry
```

Comment:
I want to bring this to the @open-telemetry/dotnet-approvers' attention.
Maybe we need to be clear about the recommended place for questions.
I would think Slack channel is **worse** than GitHub issues, because GitHub issues are easier to search and tag, folks who came later can still benefit from that. Chat is something very transient.

Reply:
I agree with the problem of a chat not being a good place to benefit others long term.
The `opentelemetry-cpp` project has [discussions open ](https://github.com/open-telemetry/opentelemetry-cpp/discussions)on their GitHub repository, so that could be another option for us to consider.